### DISCUSSION TITLE
Why .net framework for examples?

### DISCUSSION THREAD

**Discussion by chmorgan** [chmorgan/sharppcap#271] [General]
@kayoub5 do you remember why we are targeting .net framework for the example projects and not .net core? I'm trying to run the examples here and seeing an issue resolving wpcap on osx, which would make sense as .net framework 4.8 (the version we we targeting) is only in .netcore 3.0 or .net 5.0+.

Should we change examples to target .net core 3.0?

**Comment by kayoub5** [comment]
For console examples using .net core or .net framework does not make a difference, the winforms example however may not work in .net core (depending on what version of .net core you use)

**Reply by chmorgan** [reply]
Makes sense. Let me convert the examples over to .net core so they work correctly here, and on Linux where I'm guessing the native library changes may have broken them on master.
